client/service/src/client/client.rs

```diff
@@ -802,7 +802,7 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 
 		operation.op.insert_aux(aux)?;
 
-		if make_notifications {
+		if make_notifications || tree_route.is_some() {
```

Comment:
```suggestion
		// we only notify when we are already synced to the tip of the chain or if this import triggers a re-org
		if make_notifications || tree_route.is_some() {
```

Reply:
Fuck, I wanted to add a comment but than forgot it xD